### ISSUE TITLE
Software paper for report

### ISSUE THREAD

**Issue by DominiqueMakowski** [easystats/report#47]
Once `correlation` and `estimate` are on CRAN, report will go in, with a high priority to make it citeable. 

I think that it makes sense to associate all contributors of easystats to this publication, as it heavily relies on all of the easystats dependencies.

Here's a list of potential journals (feel free to complete the list):

- Software Journals:
  - [Nature toolbox](https://www.nature.com/nature/articles?type=toolbox) (not sure if possibility of submitting ourselves)
  - [JOSS](https://joss.theoj.org/) (IF = NA, H  = NA, SJR = NA)
  - [JORS](https://openresearchsoftware.metajnl.com/) (IF = NA, H = 2, SJR = NA)
  - [PeerJ Computer Science](https://peerj.com/computer-science/) (IF = NA, H = 10, SJR = 0.87)
  - [SoftwareX](https://www.journals.elsevier.com/softwarex/) (IF = NA, H =10, SJR = 4.54)
  - [Journal of Statistical Software](https://www.jstatsoft.org/index) (IF = NA, H = 115, SJR = 17.57)
- More general journals:
     - [Advances in Methods and Practices in Psychological Science](https://www.psychologicalscience.org/publications/ampps) (IF = NA, H = NA, SJR = NA)
    - [Behavior Research Methods](https://www.psychonomic.org/page/BRM) (IF = NA, H = 114, SJR = 2.69)
    - [Psychological Methods](https://www.apa.org/pubs/journals/met/index) (IF = 8.188, H = 131, SJR = 6.51)
    
    - [Scientific Reports](https://www.nature.com/srep) (IF = 4.525, H = 149, SJR = 1.41)

any opinion and experience are welcomed

**Comment by IndrajeetPatil** [comment]
I would vote for _AMPPS_.

**Comment by bwiernik** [comment]
Behavior Research Methods is also super excited about software package papers lately.

**Comment by IndrajeetPatil** [comment]
Should we start working on this? 

I think it's about time. The API is more or less stable; it's now just a matter of supporting more and more models.

**Comment by DominiqueMakowski** [comment]
I think before that issues to be fixed are:

- [ ] Solve the effectsize/parameters issue (on which one to rely)
- [ ] Communicate about how it currently works (so that all maintainers have some idea of the code logic)
- [ ] Have the code logic double (triple) checked because currently it might not be the most efficient one (it's slow, because there might be some duplicated operations)

I'll get to that at some point

**Comment by mattansb** [comment]
I also agree with @DominiqueMakowski that those issues should be resolved first.

@IndrajeetPatil If indeed the API is stable (yay!) that probably deserves a vignette - currently the docs on the pkgdown site are lots of standalone examples. Would be great to see a more cohesive look at the package's functionality.

I suggest focusing on htest objects, g/lm models and data frames at first (before moving on to mixed models, Bayes, etc...)
